src/app/app.component.spec.ts

```diff
@@ -1,12 +1,32 @@
-import { TestBed } from '@angular/core/testing';
+import { HttpClient } from "@angular/common/http";
+import { HttpClientTestingModule, HttpTestingController } from "@angular/common/http/testing";
+import { async, TestBed } from '@angular/core/testing';
+import { TranslateLoader, TranslateModule, TranslateService } from "@ngx-translate/core";
 import { AppComponent } from './app.component';
+import { HttpLoaderFactory } from "./app.module";;
+const TRANSLATIONS_EN = require('../assets/i18n/en.json');
+const TRANSLATIONS_PL = require('../assets/i18n/pl.json');
+const TRANSLATIONS_ES = require('../assets/i18n/es.json');
 
 describe('AppComponent', () => {
+  let translate: TranslateService;
+  let http: HttpTestingController;
   beforeEach(async () => {
     await TestBed.configureTestingModule({
       declarations: [
         AppComponent
       ],
+      imports: [
+        HttpClientTestingModule,
+        TranslateModule.forRoot({
+          loader: {
+            provide: TranslateLoader,
+            useFactory: HttpLoaderFactory,
+            deps: [HttpClient]
+          }
+        })
+      ],
+      providers: [TranslateService]
```

Comment:
revert changes please, do not make any changes in spec files for now, there is no point, we will do it everything at the end